packages/css/src/plugins/expand-shorthands/__tests__/flex.test.ts

```diff
@@ -127,6 +127,20 @@ describe('flex property expander', () => {
                   "
           `);
     });
+
+    it('should expand flex 0 basis', () => {
+      const result = transform`
+      flex: 1 1 0;
+    `;
+
+      expect(result).toMatchInlineSnapshot(`
+              "
+                    flex-grow: 1;
+                    flex-shrink: 1;
+                    flex-basis: 0px;
```

Comment:
I should probably try to replicate the[ issue](https://github.com/atlassian-labs/compiled/issues/1350) to understand the change, but should it be `0%` based on [previous fix](https://github.com/atlassian-labs/compiled/pull/1317/files#diff-5b489bd99f3b48b7aa7144ee849e92582c4a6e452ad02874b3216aa16ca01d73R10) or the latter is addressing a different issue?

Reply:
This is the same issue but a different manifestation, as we didn't expect the expansion to be required for three-valued flex values.

Reply:
Also I think the unit is irrelevant here but consistency would be better, will fix